packages/ui/src/pages/MyStudiesPage/MyStudiesPage.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import { useAuthStore } from "../../stores/auth";
+import { RouterLink, useRouter } from "vue-router";
+import StudyComponent from "./components/StudyComponent.vue";
+import { useQuery } from "@tanstack/vue-query";
+import studiesApi from "@/api/studies";
+
+const router = useRouter();
+const authStore = useAuthStore();
+
+if (!authStore.currentUser) {
+  router.push("/login");
+}
+
+const { data } = useQuery({
+  queryFn: studiesApi.getStudies,
+});
+
+const studies = data;
+</script>
+
+<template>
+  <div class="flex justify-center mt-14 mb-6">
+    <div class="flex flex-row w-3/4">
+      <h1 class="flex w-3/5 text-3xl font-bold">My Studies</h1>
+      <div class="flex w-2/5 justify-end">
+        <RouterLink to="/study">
+          <button
+            class="flex text-sm bg-neutral-300 border border-black rounded-lg py-1 w-30 p-4 h-8 justify-center"
```

Comment:
Wow that's a lot of flex! (And fractional widths) You definitely don't need this much styling, and it has some pretty funky effects when you make the page smaller. Here's a few tips:
* If you have an element with `display: flex` that contains a single child element, you're probably doing something wrong i.e.
```
<div class="flex">
  <div ...>
    ...
  </div>
</div>
```
* See if you can do this layout without any `w-a/b`, you don't need it!
* Think about which elements should be a constant size (`w-n`), and which ones should grow/shrink when the page size changes (`flex-1`), that's what flex is great for!

Reply:
Happy to pair as well if you want